core/backend.go

```diff
@@ -61,11 +61,11 @@ type Backend interface {
 	// ID returns the validator's ID
 	ID() []byte
 
-	// MaximumFaultyNodes returns the maximum number of faulty nodes based
+	// MaximumFaulty returns the maximum number of faulty nodes based
 	// on the validator set.
-	MaximumFaultyNodes() uint64
+	MaximumFaulty() uint64
 
-	// Quorum returns what is the quorum size for the
-	// specified block height.
-	Quorum(blockHeight uint64) uint64
+	// HasQuorum returns true if the quorum is reached
+	// for the specified block height.
+	HasQuorum(view *proto.View, messages []*proto.Message) bool
```

Comment:
Just curious: do you need the entire view here (round + block height) or just the height?

Reply:
Probably not

Reply:
I added it because seemed to be necessary for Subscriptions but in the end it was not so refactored in https://github.com/0xPolygon/go-ibft/pull/39/commits/ff0b77dd5f6949776b194e60a891d65f41fcb719